README.md

```diff
@@ -113,7 +113,7 @@ By using conditions and actions, it becomes straightforward to achieve a number
   or [aborted](https://javadoc.jenkins.io/hudson/model/Result.html#ABORTED)
   builds from the build history if they do not provide any value.
 - Keep only the last builds depending on their [result](https://javadoc.jenkins.io/hudson/model/Result.html),
-  so the history contains the most recent builds with the specified result(s): aborted, unstable, failure, or success.
+  so the history contains the most recent builds with the specified result(s): aborted, unstable, not built, failure, or success.
```

Comment:
so basically any result so maybe there is no sense to list them them all?

Reply:
Jenkins UI displays 5 build results . I though that 'not built' option was missing from the list, that is why I added it.

Reply:
I mean: add not built as you did
or
remove all listed results as this does not help because ALL results are listed anyway :)

Reply:
I added back `not built` option to the list.